src/Consumer.php

```diff
@@ -76,7 +76,7 @@ public function tick(Queue $queue, array $options = [])
             return true;
         }
 
-        $this->dispatcher->dispatch(BernardEvents::PING, new PingEvent($queue));
+        $this->dispatcher->dispatch(new PingEvent($queue), BernardEvents::PING);
```

Comment:
As per changes in version 4.x arguments  for name and value are flipped for `dispach` method signature. 

See https://github.com/symfony/event-dispatcher/blob/5.x/CHANGELOG.md